docs/vc-spec.md

```diff
@@ -183,7 +183,7 @@ After receiving the notification that II is ready, the relying party can request
     * `credentialId`: The ID of the credential that the relying party wants to request from the issuer.
   * `credentialSubject`: The subject of the credential as known to the relying party. Internet Identity will use this principal to ensure that the flow is completed using the matching identity.
 
-#### Example
+_Example_
```

Comment:
I'm not a fan of not making it a header. If it's a header, you can link to it, if it's just in italics, you cant.

Reply:
I did that as `##### Example` renders (sometimes) in the same font as the section title.  But I'm not attached to it, so I reverted this change.